bindings/matlab/README.md

```diff
@@ -0,0 +1,18 @@
+# libm2k-matlab
+[![Generic badge](https://img.shields.io/badge/MATLAB-R2020a-BLUE.svg)](https://shields.io/)
+
+MATLAB binding for the [libm2k](https://github.com/analogdevicesinc/libm2k) interface library.
+
+Documentation is available on [wiki.analog.com](https://wiki.analog.com/university/tools/m2k/matlab). The C++ API reference, which these bindings mirror, is available on [GitHub](https://analogdevicesinc.github.io/libm2k/index.html).
+
+## Releases
+
+Always install the latest release for your MATLAB version from the [releases page](https://github.com/analogdevicesinc/libm2k-matlab/releases/latest).
+
+### Latest Development Builds
+
+|  Operating System | GitHub master status  | MATLAB Release |  Installer Package  |
+|:-------:|:---------------------:|:-------:|:-------------------:|
+| Windows 10 | [![pipeline status](https://gitlab.com/tfcollins/libm2k-matlab/badges/master/pipeline.svg)](https://gitlab.com/tfcollins/libm2k-matlab/commits/master) | R2020a | <a href="https://gitlab.com/tfcollins/libm2k-matlab/-/jobs/artifacts/master/download?job=package:win64"><img src="https://upload.wikimedia.org/wikipedia/commons/2/21/Matlab_Logo.png" data-canonical-src="https://upload.wikimedia.org/wikipedia/commons/2/21/Matlab_Logo.png" height="40" /></a>|
```

Comment:
I think you can remove these badges and links.
Leave them empty for now, until we create the CI jobs and generate valid links.